### PR TITLE
feature: Add the log in/sign up form

### PR DESCRIPTION
**Pull request by RuipuCui**
feature: Add the login/sign up form, the form pops up to the centre of the page when click corresponding buttons in header.

### CODE REVIEW

**Review by YuetongLiiii — CHANGES_REQUESTED** [review]
Hi, Pls adjust the position of the words in nav bar

**Review by Leshangpang — CHANGES_REQUESTED** [review]
I think currently there is an issue with when the login/sign up window is open, the left/right buttons in the background can still be accessed. I think this is an issue that might need some look at. 
Also note that we will need to add cloudfare to the log in page for authentication, so maybe leave some space for it now (i.e. put an image as a placeholder) 
Also, I think maybe need to review the current font for login/sign up and including the highlight (boldness) to match the rest of the website. 
Ensure consistency for "log in" and "sign up" across the website. Match the spelling in window with the buttons.

File: itp-front/src/components/Login.vue
Comment (Leshangpang) [inline itp-front/src/components/Login.vue:1]: Change the X for the close button, use elements from the library. Or remove the button and close the form using another way (click outside of the form to close)

**Review by Leshangpang — CHANGES_REQUESTED** [review]
Change the close-button.

File: itp-front/src/components/Login.vue
Comment (RuipuCui) [reply itp-front/src/components/Login.vue:1]: done. I used a close button in element ui

**Review by YuetongLiiii — APPROVED** [review]
good fix. We can working on learning page

**Pull request merged**